lib/DDG/Spice/Timer.pm

```diff
@@ -5,7 +5,16 @@ use strict;
 use DDG::Spice;
 
 my @triggers = qw(timer countdown alarm);
-triggers startend => @triggers;
+# Triggers that are valid in start only 
+my @startTringgers = qw(start begin set run);
```

Comment:
I suspect `Tringgers` is a typo? Looks like its been copied throughout the file...